uui-components/src/pickers/PickerToggler.tsx

```diff
@@ -104,26 +103,38 @@ function PickerTogglerComponent<TItem, TId>(props: PickerTogglerProps<TItem, TId
 
     const renderItems = () => {
         const maxItems = getMaxItems(props.maxItems);
-        if (props.selectedRowsCount > maxItems) {
-            return props.renderItem?.({
-                value: i18n.pickerToggler.createItemValue(props.selectedRowsCount, props.entityName || ''),
-                onCheck: () => {
-                    props.onClear?.();
+        let isDisabled = props.isDisabled || props.isReadonly;
+
+        const multiItems = props.selection?.map((row) => {
+            isDisabled = isDisabled || row.isDisabled;
+
+            const newMultiItems = {
```

Comment:
Let's rename this variable to 'tagProps', to be more informative

Reply:
done